sdk/core/azure-core-http-okhttp/src/main/resources/META-INF/services/com.azure.core.implementation.http.spi.HttpClientProvider

```diff
@@ -0,0 +1 @@
+com.azure.core.http.okhttp.implementation.OkHttpClientProvider
```

Comment:
Just for my understanding, what is the purpose of this file?

Reply:
This is the configuration file that `ServiceLoader` API uses to detect `OkHttpClientProvider` as an implementation of `Service Provider`. More info can be found [here](https://docs.oracle.com/javase/7/docs/api/java/util/ServiceLoader.html).